hkmc2/shared/src/main/scala/hkmc2/codegen/HandlerLowering.scala

```diff
@@ -345,34 +346,33 @@ class HandlerLowering(using TL, Raise, Elaborator.State, Elaborator.Ctx):
       else None
     ).collect { case Some(b) => b }
 
-    val fnBmsMap = funDefns.map {
-      case sym: BlockMemberSymbol => 
-        sym -> BlockMemberSymbol(sym.nme + "$" + thirdPassFresh(), sym.trees)
-    }.toMap
+    val fnBmsMap = funDefns.map(b =>
+      b -> BlockMemberSymbol(b.nme + "$" + thirdPassFresh(), b.trees)
+    ).toMap
```

Comment:
The RHS of such definitions should be indented (same remark for the other similar places in this file).

Reply:
So you mean like
```fs
    val fnBmsMap = funDefns.map(b =>
        b -> BlockMemberSymbol(b.nme + "$" + thirdPassFresh(), b.trees)
      ).toMap
```
?

Reply:
Yes, or
```scala
    val fnBmsMap = funDefns.map: b =>
        b -> BlockMemberSymbol(b.nme + "$" + thirdPassFresh(), b.trees)
      .toMap
```
if you want.